Assets/Scripts/CutsceneCamera.cs

```diff
@@ -6,41 +7,72 @@ public class CutsceneCamera : MonoBehaviour
 {
     [CanBeNull] private static GameObject _targetObject;
     private static Vector2 _target;
+    private static Vector2 _lead;
+    private static Camera _camera;
+    private static float _cameraResizeSpeed;
+    private static float _cameraSize;
 
     [SerializeField] private float xSmoothTime;
     [SerializeField] private float ySmoothTime;
     private float _xVelocity;
     private float _yVelocity;
 
+    private void Awake()
+    {
+        _camera = GetComponent<Camera>();
```

Comment:
nit: could add `[RequireComponent(typeof(Camera))]` as a class attribute since you don't check `_camera` for null